stl/inc/limits

```diff
@@ -12,6 +12,9 @@
 #include <climits>
 #include <cwchar>
 #include <intrin0.h>
+#if defined(_M_ARM64) && !defined(_M_ARM64EC) // TRANSITION, GH-2129
+#include <arm64_neon.h>
+#endif // defined(_M_ARM64) && !defined(_M_ARM64EC)

```

Comment:
Style nitpick: This isn't sorted, and clang-format won't move headers across an `#if/#endif`. Can we move this to the end, i.e. below `#include <xstddef>`?